www/include/configuration/configObject/meta_service/listMetaService.php

```diff
@@ -97,17 +97,37 @@
 /*
  * Meta Service list
  */
+$conditionStr = "";
+$metaStrParams = [];
+//binding query params for non admin  acl rules
+if (!$acl->admin && $metaStr) {
+    $metaStrList = explode(',', $metaStr);
+    foreach ($metaStrList as $index => $meta_id) {
+        $metaStrParams[':meta_' . $index] = $meta_id;
+    }
```

Comment:
No snake_case, only CamelCase.
```suggestion
    foreach ($metaStrList as $index => $metaId) {
        $metaStrParams[':meta_' . $index] = (int) $metaId;
    }
```